### PR TITLE
Allow (limited) guest commanders

### PR DESCRIPTION
**Pull request by jaj22**
### What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [X] Enhancement

### What have you changed and why?
Problem: Not having a commander makes it impossible to play Antistasi properly. This is more important with the attack/support rework, because territory loss is no longer backstopped by the mission radius check, and HQ attacks may be ordered repeatedly once the HQ is discovered.

Solution: Added a parameter and supporting code to allow guest commanders, with members given strictly higher priority. Members can force a re-election when there's a guest commander by toggling their eligibility status on.

Guest commanders are restricted to commander functionality that involves moving HQ, high command and spending faction money. Functions that were already member-specific (eg. arsenal access, recruiting AI) stay member-specific, while some commander functions are not available to guest commanders:
- Garage unlocking override
- Arsenal limits dialog
- Stealing money from the faction

Guests now have their rank & score loaded from save, which should help identify suitable commanders.

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.
1. [X] Have you loaded the mission in LAN host?
2. [X] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [X] No
2. [ ] Yes (Please provide further detail below.)

There's some old weirdness in the garage code that I might fix, although it's not strictly required: Having an admin logged in prevents the commander from unlocking vehicles in the garage.

### How can the changes be tested?
Pain. Terrible pain.

### CODE REVIEW

File: A3A/addons/Garage/Public/config.inc [hunk 1]
```diff
@@ -19,7 +19,7 @@
 HR_GRG_Prefix = "Antistasi";
 //Overwrite client
 HR_GRG_cmdClient = {
-    if ((isServer && hasInterface) || {admin owner _this > 0}) then {_this} else {theBoss}; //LAN Host or admin on DS or commander
+    if ((isServer && hasInterface) || {admin owner _this > 0}) then {_this} else { [objNull, theBoss] select (theBoss call A3A_fnc_isMember) }; //LAN Host or admin on DS or member-commander
```
Comment (HakonRydland) [inline A3A/addons/Garage/Public/config.inc:22]: update to current unstable

File: A3A/addons/core/functions/init/fn_initClient.sqf [hunk 2]
```diff
@@ -94,6 +89,7 @@ player setVariable ["rankX",rank player];
 
 player setVariable ["owner",player,true];
 player setVariable ["punish",0,true];
+player setVariable ["playerUID",getPlayerUID player,true];
```
Comment (HakonRydland) [inline A3A/addons/core/functions/init/fn_initClient.sqf:92]: we should really be prefixing variables on objects

**Review by HakonRydland — CHANGES_REQUESTED** [review]
two minor things then this can be merged

File: A3A/addons/Garage/Public/config.inc (hunk 1 above)
Comment (jaj22) [reply A3A/addons/Garage/Public/config.inc]: Done, fixed the comment while I was there.

File: A3A/addons/core/functions/init/fn_initClient.sqf (hunk 2 above)
Comment (jaj22) [reply A3A/addons/core/functions/init/fn_initClient.sqf]: Done.

**Review by HakonRydland — APPROVED** [review]

**Pull request merged**